packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -83,11 +89,24 @@ export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
             );
             reject(err);
           }
-          resolve();
+          resolve(void 0);
         });
       });
     }
   }
+
+  public logMetric(
```

Comment:
Where is this method called during creation? I'm thinking if there is any other way to readily get the language information.

Reply:
I was struggling with this one on Thursday, and this was the suggestion of @brpowell. It gets called further downstream in the extended class BaseTemplateCommand's execute, which here we aren't overriding in forceFunctionCreate.

Reply:
It's to attach the language property to the `commandExecution` event that already gets called in `BaseTemplateCommand` for this command and not send additional or new events

Reply:
Got it! One more question - if we are going with this.getFileExtension(), the language would be js, ts or java correct? Whereas it is node or java in other places that uses FunctionService's getFunctionLanguage.

Reply:
@randi274 - Just read your comment on the story and you talk about language being java, javascript or typescript. Can we have consensus on what it is going to be? We should be fine with java, ts or js (Based on the file extension, if that's the easiest) as long it is consistent everywhere.

Reply:
Ah right, that's a great point @jag-sfdc! I lost focus on that when we were talking through this on Thursday. Let me think on making that consistent, and then get back to you if I'm unclear on how to make that happen.

Reply:
It should be fine just to make it the file extension, unless we can think of a function that is two different languages with the same extension.